2-Webpack/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  mode: "development",
+  entry: {
+    main: path.resolve(__dirname, "main.js"),
+  },
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: "[name].[contenthash].js",
+    assetModuleFilename: "[name][ext]",
```

Comment:
Actually for this application it's not relevant, so I will remove it.
Basically what it does, it saves the original name of any file (e.g a logo.svg) instead of hashing it.